src/PlaywrightSharp.Chromium/ChromiumBrowser.cs

```diff
@@ -145,15 +145,15 @@ private async void Session_MessageReceived(object sender, MessageEventArgs e)
                 switch (e.MessageID)
                 {
                     case "Target.targetCreated":
-                        await CreateTargetAsync(e.MessageData?.ToObject<TargetCreatedResponse>()).ConfigureAwait(false);
+                        await CreateTargetAsync(e.MessageData?.ToObject<TargetTargetCreatedEventArgs>()).ConfigureAwait(false);

```

Comment:
I don't like the `EventArgs` suffix for these classes.

Reply:
hmm, these classes should be used instead in the event handler.
We shouldn't need to create POCO objects, maybe 1-2 and that's it. everything else is covered in the generated protocol classes

Reply:
Our events don't necessarily match browser events, in fact, those are at the Abstraction layer, which means that our events are cross-browser events.
e.g. [FileChooserEventArgs](https://github.com/kblok/playwright-sharp/blob/master/src/PlaywrightSharp.Abstractions/FileChooserEventArgs.cs#L8) that's a generic, cross-browser event we need to build. It's **cross-browser**, I think is a key concept. 

In this example, this is clearly not an event args. It's not an event we get from an event handler and it's not an argument we are going to send using an event handler.
Chromium might have called it event, but for us, it's just a message.

Reply:
Ahh, Makes perfect sense.
So maybe changing the suffix of the chromium event classes to `ChromiumEvent` is better?

Reply:
Yeah let's do it!